site/docs/spec/specification.md

```diff
@@ -2,41 +2,38 @@
-|          | sketch | [Physical Relations](/relations/physical_relations)          | Specific execution sub-variations of common relational operations that describe have multiple unique physical variants associated with a single logical operation. Examples include hash join, merge join, nested loop join, etc. |
-|          | empty  | [User Defined Relations](/relations/user_defined_relations)  | Installed and reusable relational operations customized to a particular platform. |
-|          | empty  | [Embedded Relations](/relations/embedded_relations)          | Relational operations where plans contain the "machine code" to directly execute the necessary operations. |
-| 5        | sketch | [Text Serialization](/serialization/text_serialization)      | A human producible & consumable representation of the plan specification. |
-| 6        | sketch | [Binary Serialization](/serialization/binary_serialization)  | A high performance & compact binary representation of the plan specification. |
-
+## Status
+
+The specification has passed the initial design phase and is now in the final stages of being fleshed out.  The community is encouraged to identify (and address) any perceived gaps in functionality using GitHub issues and PRs.
+
+
+## Components (Complete)
+
+| Section                                                      | Description                                                  |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Simple Types](/types/type_classes#simple-types)             | A way to describe the set of basic types that will be operated on within a plan. Only includes simple types such as integers and doubles (nothing configurable or compound). |
+| [Compound Types](/types/type_classes#compound-types)         | Expression of types that go beyond simple scalar values. Key concepts here include: configurable types such as fixed length and numeric types as well as compound types such as structs, maps, lists, etc. |
+| [Type Variations](/types/type_variations)                    | Physical variations to base types.                           |
+| [User Defined Types](/types/type_classes#user-defined-types) | Extensions that can be defined for specific IR producers/consumers. |
+| [Field References](/expressions/field_references)            | Expressions to identify which portions of a record should be operated on.     |
+| [Scalar Functions](/expressions/scalar_functions)            | Description of how functions are specified. Concepts include arguments, variadic functions, output type derivation, etc. |
+| [Scalar Function List](https://github.com/substrait-io/substrait/blob/main/extensions/scalar_functions.yaml) | A list of well-known canonical functions in YAML format.     |
```

Comment:
https://github.com/substrait-io/substrait/tree/main/extensions will get us to the list of YAML files.  The blob file listed here leads to a 404 page.